src/form/input-wrapper.js

```diff
@@ -101,39 +102,10 @@ export class InputWrapper extends Component {
     return render({
       value: this.formatNumber(),
       update: updateFormModel.bind(null, name, type || 'text'),
-      validate: validateField.bind(null, name),
       error: errors[name],
-      isValidating: validating[name],
-      formDisabled,
-      updateFormModel,
-      validateField,
-      showRealValue: this.showRealValue,
-      showPrettyValueAndValidate: this.showPrettyValueAndValidate
-    })
-  }
-}
-
-export class SubmitWrapper extends Component {
-  static propTypes = {
-    render: func.isRequired,
-  }
-
-  static contextTypes = {
-    form: object.isRequired,
-    formActions: object.isRequired,
-  }
-
-  render() {
-    const {
-      form: { formDisabled, formValidating, formValid },
-      formActions: { submitForm },
-    } = this.context
-
-    return this.props.render({
-      formDisabled,
-      formValidating,
-      formValid,
-      submitForm,
+      inputDisabled: validating[name] || formDisabled,
+      onFocus: this.showRealValue,
+      onBlur: this.showPrettyValueAndValidate
```

Comment:
i think i agree with changing the names of these although i think we maybe shouldn't be opinionated by suggesting `onFocus` and `onBlur` -- i can see situations where these might be handled in slightly different circumstances and the names would be confusing. we should instead suggest in the documentation that `showRealValue` or whatever we decided to call it should be mapped to `onFocus` on the input element in most cases.